doc/userguide/rules/base64-keywords.rst

```diff
@@ -27,6 +27,10 @@ This option makes ``offset`` skip bytes relative to the previous match.
 
     The content match that you want to decode relative to must be the first match in the stream.
 
+.. note:: ``base64_decode`` follows RFC 4648 by default i.e. encounter with any character that is not found in the base64 alphabet leads to rejection of that character and the rest of the string.
```

Comment:
Does it make sense to have this info as a note @jufajardini ?

Reply:
I had thought of something like that, indeed!